toolkit/tools/imageconfigvalidator/imageconfigvalidator.go

```diff
@@ -135,28 +134,30 @@ func validatePackages(config configuration.Config) (err error) {
 			selinuxPkgName = configuration.SELinuxPolicyDefault
 		}
 
-		for _, pkg := range packageList {
-			// The installer tools have an undocumented feature which can support both "pkg-name" and "pkg-name=version" formats.
-			// This is in use, so we need to handle pinned versions in this check. Technically, 'tdnf' also supports "pkg-name-version" format,
-			// but it is not easily distinguishable from "long-package-name" format so it will not be supported here.
-			pkgVer, err := pkgjson.PackageStringToPackageVer(pkg)
-			if err != nil {
-				return fmt.Errorf("%s: %w", validateError, err)
-			}
+		foundKernelPackage, err := installutils.PackagelistContainsPackage(packageList, kernelPkgName)
+		if err != nil {
+			return fmt.Errorf("%s: %w", validateError, err)
+		}
 
-			if pkgVer.Name == kernelPkgName {
-				return fmt.Errorf("%s: kernel should not be included in a package list, add via config file's [KernelOptions] entry", validateError)
-			}
-			if pkgVer.Name == dracutFipsPkgName {
-				foundDracutFipsPackage = true
-			}
-			if pkgVer.Name == selinuxPkgName {
-				foundSELinuxPackage = true
-			}
-			if pkgVer.Name == userAddPkgName {
-				foundUserAddPackage = true
-			}
+		foundDracutFipsPackage, err = installutils.PackagelistContainsPackage(packageList, dracutFipsPkgName)
+		if err != nil {
+			return fmt.Errorf("%s: %w", validateError, err)
```

Comment:
Since we're here: do we still do `\n` for errors?
```suggestion
			return fmt.Errorf("%s:\n%w", validateError, err)
```

Reply:
This tool hasn't ever used them, was just keeping consistent.